components/atoms/CreativeCommonsNotice/index.tsx

```diff
@@ -13,11 +13,19 @@ const CreativeCommonsNotice: React.FunctionComponent = () => (
     <StyledCreativeCommonsNoticeContainer>
         <StyledCreativeCommonsNoticeText>
             This work is licensed under a{' '}
-            <StyledCreativeCommonsLink href={creativeCommonsUrl}>
+            <StyledCreativeCommonsLink 
+                href={creativeCommonsUrl}
+                target="_blank"
```

Comment:
This isn't related to this PR, but was something I meant to wrap up in the CSS reorg.